dm/master/server.go

```diff
@@ -1322,6 +1322,28 @@ func (s *Server) OperateSource(ctx context.Context, req *pb.OperateSourceRequest
 	return resp, nil
 }
 
+// OperateLeader implements MasterServer.OperateLeader
+// Note: this request dosen't need to forward to leader
```

Comment:
You mean this command **always** takes effect on the current member which is handling the request?

If we add multiple DM-master endpoints support on dmctl (and other clients), this may become a problem because the client may choose a random DM-master to send the request.

Reply:
yes, which master receives this request then takes effect on itself.  If we need to evict by master name, then every master needs to maintain the link to other masters, this looks not well.